apps/users/management/commands/migrate_2fa_data.py

```
@@ -0,0 +1,179 @@
+            cursor.execute(
+                """
+                SELECT t.token
+                FROM otp_static_statictoken t
+                JOIN otp_static_staticdevice d ON d.id = t.device_id
+                WHERE d.user_id = %s AND d.confirmed = true
+            """,
+                [user_id],
+            )
+            return [row[0] for row in cursor.fetchall()]
+
+    def handle(self, *args, **options):
```

Ruff (0.14.10)

Unused method argument: `args`

(ARG002)